mf2py/dom_helpers.py

```diff
@@ -51,6 +51,20 @@ def get_attr(el, attr, check_name=None):
     if isinstance(check_name, (tuple, list)) and el.name in check_name:
         return el.get(attr)
 
+def parse_srcset(srcset, base_url=''):
+    result = {}
+
+    srcset = srcset.split(',')
+
+    for src in srcset:
+        src = src.strip()
+        if not src:
+            continue
+
+        src, size = src.split(' ', 1)
+        result[size] = try_urljoin(base_url, src)
```

Comment:
Minor, but when a size (condition) is repeated, it looks like this will take the last element with that size in the srcset list. https://github.com/microformats/microformats2-parsing/issues/7#issuecomment-1257104515 says to take the first element instead. Maybe not a big deal either way?

Reply:
> https://github.com/microformats/microformats2-parsing/issues/7#issuecomment-1257104515 says to take the first element instead. Maybe not a big deal either way?

Correct! The proposed specification in my comment (that @snarfed linked above) was derived from existing specifications from Mozilla and the WHATWG. Basically, "first one wins" in cases where the size descriptor (explicit or implicit as with the default `1x`) is repeated in the markup.